docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/04_inheritance.md

```diff
@@ -346,24 +356,21 @@ typically only be done for conceptual purposes.
 
 To use unions, turn on the `useUnionQueries` setting.
 
+**app/_graphql/config.yml**
 ```yaml
-SilverStripe\GraphQL\Schema\Schema:
-  schemas:
-    default:
-      config:
-        modelConfig:
-          DataObject:
-            plugins:
-              inheritance:
-                useUnionQueries: true
+modelConfig:
```

Comment:
What are we delete a bunch of the yml examle, I find the original more intuitive to follow

Reply:
The `config.yml` file gets automatically scaffolded as `SilverStripe\GraphQL\Schema\Schema.schemas.default.config` so having that there would actually cause errors.